### DISCUSSION TITLE
🎉 New Release: Fixes for Android + Open in New Window, Property: None (v1.6)

### DISCUSSION THREAD

**Discussion by chrisgurney** [chrisgurney/obsidian-note-toolbar#54] [Announcements]
# Note Toolbar v1.6 is now available in Community Plugins!

[Note Toolbar](https://github.com/chrisgurney/obsidian-note-toolbar) v1.6 has been released, primarily to fix to important issues (see below). I opted to release this as a major release due to a fundamental change to how links are rendered (from `a` to `span` tags), which had the potential to break things like the `Focus` command and accessibility in general.

### Installation

Update via Community Plugins, or search for _Note Toolbar_ there and install.

## Migration Notes

- To enable the following use case, [variables](https://github.com/chrisgurney/obsidian-note-toolbar/wiki/Variables) substituted into URIs are no longer encoded. I may need to add an option for this in the future; feedback would be appreciated.
  - You can now put a fully-formed URL (i.e., one that includes the protocol) in a property and refer to it in a URI item using [variables](https://github.com/chrisgurney/obsidian-note-toolbar/wiki/Variables), and be able to have it open the website without issue.

## New Features

- Set a note's `notetoolbar` property to `none` to completely remove the toolbar from that note. (Thanks @caglarongan)

## Improvements

- Folder Mappings: Adds suggestions for `*` and `/` to the folder dropdown list
- Style Settings plugin: You can now override font and icon sizes. (Thanks @caglarongan for the idea!)

## Fixes

- Fixes the issue with links crashing on Android (reading mode) ([#19](https://github.com/chrisgurney/obsidian-note-toolbar/issues/19))
- Fixes issue with toolbars not displaying correctly when notes are opened in a new window ([#46](https://github.com/chrisgurney/obsidian-note-toolbar/issues/46))
- Fixes to settings keyboard navigation (e.g., backspace = delete)

## Changelog

- fix: open in new window: TBs not showing correctly ([#46](https://github.com/chrisgurney/obsidian-note-toolbar/issues/46)) [↗](https://github.com/chrisgurney/obsidian-note-toolbar/commit/27f3dd19cd3f71f238437ac50f7bbbd6ab401716)
- feat: added font and icon sizes to Style Settings support [↗](https://github.com/chrisgurney/obsidian-note-toolbar/commit/df36ab9e18e13abbee3f831b422e3544bc6cc494)
- fix: turn off uri encoding to allow more flexibility [↗](https://github.com/chrisgurney/obsidian-note-toolbar/commit/a2d4bdb22f70748d3755627591c10d53cb8bedcc)
- docs: added instructions at bottom of IconSuggester [↗](https://github.com/chrisgurney/obsidian-note-toolbar/commit/6872b071309d921376269b3bbcf5162c9dd81826)
- style: replaced cross with trash icon [↗](https://github.com/chrisgurney/obsidian-note-toolbar/commit/1a8b2f893d96365b8a6ce1ef89cbcebebdd47e7c) [↗](https://github.com/chrisgurney/obsidian-note-toolbar/commit/82ff3981b0376eb46c646803873a242b806525f6)
- fix: (attempt) for links crashing Android ([#19](https://github.com/chrisgurney/obsidian-note-toolbar/issues/19)) [↗](https://github.com/chrisgurney/obsidian-note-toolbar/commit/ccddb50625613f662246db81f76924743e97cb83)
- fix: add enter to keyboard handler [↗](https://github.com/chrisgurney/obsidian-note-toolbar/commit/b1bb020de7b4aa7523955c678d056fa932035852)
- feat: set note toolbar prop to 'none' to not display a toolbar on that note [↗](https://github.com/chrisgurney/obsidian-note-toolbar/commit/cba1e61ed5a770e5277fb8b7c276404a06afd8c4)
- chore: added pattern label [↗](https://github.com/chrisgurney/obsidian-note-toolbar/commit/e06af0322164920ceeb41dbfbb61f7f9925deac2)
- style: focus effect for toolbar and item lists [↗](https://github.com/chrisgurney/obsidian-note-toolbar/commit/e2ca5cd702b0ebc1346cf4d6ceca59bc33ae0a83)
- fix: list handler: backspace key should delete; space jumping screen [↗](https://github.com/chrisgurney/obsidian-note-toolbar/commit/0bdd9f80e89290745106088553192b346812c56a)
- feat: add suggestions for * and / to folder mapping [↗](https://github.com/chrisgurney/obsidian-note-toolbar/commit/dcd00289ab94696d652252fc5eaaa9aacd239444) [↗](https://github.com/chrisgurney/obsidian-note-toolbar/commit/b88a089d3d9030901fccd7f7f9da328f5730caff)

**Comment by MrComputron** [comment]
@chrisgurney Loving the changes so far! Root and all notes drop-down is much appreciated.
